build-tasks/create-rule.js

```diff
@@ -0,0 +1,78 @@
+const common = require('./common');
```

Comment:
> const common = require('./common');

I'm not a big fan of `import * as` exports for collections of miscellaneous functions. It becomes a bit harder to tease them apart & refactor when they're prefixed with a name like that. Let's switch to `const { specificOneIWant, otherFunction ] = require('./common');`.